rustls/src/vecbuf.rs

```diff
@@ -14,13 +14,23 @@ use crate::msgs::message::OutboundChunks;
 /// This avoids extra copies when appending a new byte vector,
 /// at the expense of more complexity when reading out.
 pub(crate) struct ChunkVecBuffer {
+    /// How many bytes have been consumed in the first chunk.
+    ///
+    /// Invariant: zero if `chunks.is_empty()`
+    /// Invariant: 0 <= `prefix_used` < `chunks[0].len()`
+    prefix_used: usize,
+
+    /// Subsequent chunks.
```

Comment:
Nit: the `Subsequent` here feels a little misleading to me, given that this actually contains all the chunks.

Reply:
Ah, good spot! Leftover from when I had the first item separate from `chunks`